src/Common/Factory/AbstractFactoryRule.php

```diff
@@ -13,33 +13,55 @@
 
 abstract class AbstractFactoryRule extends SprykerAbstractRule
 {
-    public const PATTERN_FACTORY = '/\\\\*\\\\.+\\\\.+\\\\[A-Za-z0-9]+(Business|Service|Communication)Factory$/';
+    public const PATTERN_FACTORY = '/\\\\*\\\\.+\\\\.+\\\\[A-Za-z0-9]+(Business|Service|Communication|Persistence)Factory$/';
+    public const PATTERN_FACTORY_EXCEPT_PERSISTENCE = '/\\\\*\\\\.+\\\\.+\\\\[A-Za-z0-9]+(Business|Service|Communication)Factory$/';
```

Comment:
Don't change it now, but if more specific cases start to get introduced I'd highly advise using functions to build up this string via composition.